content/reference/user/organization/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Organization"
+description: "Learn how to access organization details."
+date: 2021-08-04T14:29:04-00:00
+lastmod: 2021-08-04T14:29:04-00:00
+weight: 10081
+---
+
+To consult information regarding an organization, click on the **Organization name link**.
+
+{{< img src="organization-page-link.png" alt="Organization page link" >}}
+
+## Profile information
+
+{{< img src="organization-profile.png" alt="Organization profile information" >}}
+
+* **Avatar** - Compose by default from the two first characters of your **Organization name**.
+* **Organization Name** - Display name for your organization.
+* **Organization Slug** - Unique string name.
+
+
+{{< alert tip >}}
+Click on the pen icon to edit the **Organization name**.
+{{< /alert >}}
+
+## Credit consumption
+
+{{< img src="organization-credit.png" alt="Organization credit informations" >}}
+
+* **Blue** - Available credits for the current month.
```

Comment:
Both can be "for the current month" or can be "during the current month"

Reply:
This is misleading, as it may indicates that plans resets at the first day of the month.

Reply:
Done